great_expectations/data_context/store/tuple_store_backend.py

```diff
@@ -257,7 +257,24 @@ def list_keys(self, prefix=()):
                     key_list.append(key)
 
         return key_list
+   
     
+    def crmdir(self, mroot, curpath):
+        try:
+            while not os.listdir(curpath) and os.path.exists(curpath):
+                d2=curpath
+                f2=os.path.dirname(d2)
+                if os.path.isfile(curpath):
+                    os.remove(curpath)
+                else: 
+                    if os.path.isdir(curpath):
+                        shutil.rmtree(curpath)
+                curpath=f2
+                self.crmdir(mroot,curpath)
+        except (NotADirectoryError, FileNotFoundError):
+            pass
+
+
```

Comment:
This may be right, but I don't understand it. Can you provide a description? What does "crmdir" stand for?

Reply:
just removing recursively, suggest a different function name. I added description